src/app/personside/visittkort/body/vergemal/Vergemal.tsx

```diff
@@ -52,8 +52,8 @@ function Verge(props: {verge: Verge}) {
 }
 
 function Vergemal(props: {vergemal: Vergemal}) {
-    const verger = props.vergemal.verger.map(verge =>
-        <Verge verge={verge} key={verge.ident}/>);
+    const verger = props.vergemal.verger.map((verge, index) =>
+        <Verge verge={verge} key={index}/>);
```

Comment:
er vel ansett som nono å bruke index som key?

Reply:
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Jau, men greia her er at jeg ikke har noen unik id på vergene (fordi ident er optional) som jeg kan bruke som key. I artiklen du linker står det lenger ned:

>Update: Exception from the rule
Many people asked if they always, always have to generate ids. Others have suggested use cases when using the index as a key seems justifiable.

>It is true that sometimes generating new ids is redundant and may be avoided. For example translation of license terms or list of contributors.

>To help you decide, I put together three conditions which these examples have in common:

> * the list and items are static–they are not computed and do not change;
> * the items in the list have no ids;
> * the list is never reordered or filtered.
>When all of them are met, you may safely use the index as a key. 

Tror vi kan ✅ av de tre punktene :) Eller hva tenker du?